examples/demo_2d_swe.py

```diff
@@ -0,0 +1,54 @@
+
+from __future__ import division  # Get proper divison
+
+import math
+import random
+
+import numpy as np
+
+
+from firedrake import *
+
+from flooddrake import *
+
+""" demo file for simple 2d shallow water equations """
+
+
+# Meshsize
+
+n = 10
+mesh = UnitSquareMesh(n, n)
+
+# mixed function space
+X = FunctionSpace(mesh, "DG", 1)
+Y = FunctionSpace(mesh, "DG", 1)
+Z = FunctionSpace(mesh, "DG", 1)
+V = X * Y * Z
```

Comment:
Ah, fine then. The names `X`, `Y`, `Z` made me think these are just directions. Maybe they could have better names then?